scripts/netContain/build_image.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+BUILD_ENV_IMAGE_NAME="contiv-netplugin-build"
+TEMPORARY_CONTAINER_NAME="contiv-netplugin-build-temporary"
+FINAL_IMAGE_NAME="contivbase"
+
+NETPLUGIN_BINARIES="netplugin netmaster netctl contivk8s"
+
+function get_image_id() {
+	docker inspect --format '{{.ID}}' $1
+	return $?
+}
+
+function build_image_from_path() {
```

Comment:
This function has been rewritten. It was ignoring the best practices of using `docker build`. `docker build` accepts a `-t` parameter to tag an image. There's no need to have complex code to figure out if the build process has an image ID in the output or if it's some kind of error.

The exit codes of the commands was ignored before. This wasn't particularly ideal.